src/components/Components.tsx

```diff
@@ -1,13 +1,42 @@
 // React imports
 import React from "react";
 import { ButtonProps } from "../lib/types";
+import { NavLink } from "react-router-dom";
+import "./stylesheets/StylizedLink.css";
 
-const Button = ({ children, ...other }: ButtonProps) => {
+export const Button = ({ children, ...other }: ButtonProps) => {
   return (
     <button type="button" {...other}>
       {children}
     </button>
   );
 };
 
-export default Button;
+export const StylizedLink = ({
+  to,
+  title,
+  end,
+  children,
+}: {
+  to: string;
+  title?: string;
+  end?: boolean;
+  children?: React.ReactNode;
+}) => {
+  const applyActiveLinkStyling = ({
+    isActive,
+  }: {
+    isActive: boolean;
+  }): string => (isActive ? "active-stylized-link" : "");
+
+  return (
+    <NavLink
+      to={to}
+      className={applyActiveLinkStyling}
+      title={title}
+      end={end ?? false}
+    >
+      {children}
+    </NavLink>
+  );
+};
```

Comment:
Actually, it would be better to create a separate file, e.g. `/src/components/StylizedLink.tsx`.

Also, this file should never be named in this way, could you rename it to `src/components/Button.tsx` (and change it back to export default)?